model-engine/model_engine_server/common/tokenizer_utils.py

```diff
@@ -0,0 +1,173 @@
+from collections import namedtuple
```

Comment:
Would like to avoid a "common" / "utils" file as that starts to erode the clean architecture setup we have.

Reply:
Also discussing offline - IMO I think we should have a Repository for the tokenizers, and have the compute that uses the tokenizers inside the inference-related usecase.